be/src/vec/runtime/shared_hash_table_controller.h

```diff
@@ -38,36 +38,31 @@ namespace vectorized {
 class VExprContext;
 
 struct SharedHashTableEntry {
-    SharedHashTableEntry(Status status_, void* hash_table_ptr_, std::vector<Block>* blocks_,
+    SharedHashTableEntry(Status status_, void* hash_table_ptr_, const std::vector<Block>& blocks_,
                          RuntimeFilterSlotsBase<VExprContext>* runtime_filter_slots_)
             : status(status_),
               hash_table_ptr(hash_table_ptr_),
               blocks(blocks_),
               runtime_filter_slots(runtime_filter_slots_) {}
-    SharedHashTableEntry(SharedHashTableEntry&& entry)
-            : status(entry.status),
-              hash_table_ptr(entry.hash_table_ptr),
-              blocks(entry.blocks),
-              runtime_filter_slots(entry.runtime_filter_slots) {}
 
     static SharedHashTableEntry empty_entry_with_status(const Status& status) {
-        return SharedHashTableEntry(status, nullptr, nullptr, nullptr);
+        return SharedHashTableEntry(status, nullptr, {}, nullptr);
     }
 
     Status status;
     void* hash_table_ptr;
```

Comment:
lost shared hash table and shared arena?

Reply:
'emtpy entry' just contains status.